app/javascript/availability/libraries_locations.json

```diff
@@ -526,7 +533,7 @@
     "REF-FE": "Fayette - Reference Collection - In-Library Use",
     "REF-GV": "Great Valley - Reference Collection - In-Library Use",
     "REF-HB": "Harrisburg - Reference - 1st Floor - In-Library Use",
-    "REF-HN": "Copy unavailable, request via Interlibrary Loan",
+    "REF-HN": "Hazleton - Reference Collection - In-Library Use",
```

Comment:
same for this location, it is in the ill list and the label change here won't have an effect.

Reply:
now moved out of ILL list